### DISCUSSION TITLE
[Vote] Obsidian Gems of the Year 2021

### DISCUSSION THREAD

**Discussion by FelipeRearden** [chhoumann/quickadd#156] [Show and tell]
Hello friends !!!!!!!

Let’s not forget to show our love and appreciation and gratitude for our friend @chhoumann by voting for QuickAdd in the Plugin Category of the Obsidian Gems of the Year 2021.

It is the least that we can do for @chhoumann for making this AMAZING and UNIQUE plugin that makes our Obsidian lives better every day !!!!!!!!

Let’s vote right now !!!!!!!!!!!!!!!!!!!!!!!

https://forum.obsidian.md/t/obsidian-gems-of-the-year-2021-voting/28759

**Comment by chhoumann** [comment]
Thank you very much, @FelipeRearden!
